### DISCUSSION TITLE
Please Provide A Guide on How to disable real time evaluaion to be off

### DISCUSSION THREAD

**Discussion by mjishnu** [Almenon/AREPL-vscode#426] [Q&A] [answered]
i am new to vs code can you tell me steps to turn off real time evaluation

**Comment by Almenon** [comment, marked as answer]
1. read https://code.visualstudio.com/docs/getstarted/settings
2. go to settings
3. Change `AREPL.whenToExecute` to your preference (onSave or onKeybinding)

Note that the keybinding for executing arepl when you have onKeybinding set is `Ctrl+Shift+;` if you're on windows, or `cmd+shift+;` if you're on mac.
Also note the keybinding for saving the document is control+s or cmd+s, but you probably know that already.
